Document the OpenTelemetry Tracer object in the context of the serverless runtimes (#13495)
kyma-project/kyma: docs/05-technical-reference/svls-08-function-specification.md

| **ce-id** | Unique identifier of the event |
| **ce-time** | Time at which the event was sent |
| **data** | Either JSON or a string, depending on the request type. Read more about [Buffer](https://nodejs.org/api/buffer.html) in Node.js and [bytes literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) in Python. |

> | **tracer** | Fully configured OpenTelemetry [tracer](https://opentelemetry.io/docs/reference/specification/trace/api/#tracer) object that allows to communicate with the Jaeger service to share tracing data. For more informations how to use the tracer object look [here](../03-tutorials/00-serverless/svls-12-use-opentelemetry-client.md) |

Reviewer: ```suggestion
| **tracer** | Fully configured OpenTelemetry [tracer](https://opentelemetry.io/docs/reference/specification/trace/api/#tracer) object that allows you to communicate with the Jaeger service to share tracing data. For more information on how to use the tracer object see [Use the OpenTelemetry standard](../03-tutorials/00-serverless/svls-12-use-opentelemetry-client.md) |
```
